src/feat/wave-reader.cc

```diff
@@ -357,7 +357,7 @@ void WaveData::Write(std::ostream &os) const {
   int num_clipped = 0;
   for (int32 i = 0; i < num_samp; i++) {
     for (int32 j = 0; j < num_chan; j++) {
-      int32 elem = static_cast<int32>(trunc(data_ptr[j * stride + i]));
```

Comment:
Microsoft is truly awful, both C++ and their new things like UWP or async/await. Don't even think about C++11 please.